cmd/kube-apiserver/app/options/options.go

```diff
@@ -109,7 +108,6 @@ func NewServerRunOptions() *ServerRunOptions {
 		Logs:                    logs.NewOptions(),
 		Traces:                  genericoptions.NewTracingOptions(),
 
-		EnableLogsHandler:                 true,
```

Comment:
I think this PR should essentially reduce to changing this line from true to false